spec/models/object_event_hook_config_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/master/LICENSE
+require 'rails_helper'
+
+RSpec.describe ObjectEventHookConfig, type: :model do
+  describe '.webhook' do
+    before do
+      @nonprofit = create(:nm_justice)
+      @open_fn_config = create(:open_fn_config)
```

Comment:
I usually prefer `let`, but since I'm learning a new codebase, I tried to follow some that I thought could be good examples. For this one, I looked at the nonprofit spec!

Reply:
Ha, ya, that's not a good example sadly :)